projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 13.5-a.3 - 2024-05-20
+### Enhancements
+- AI Assistant: Enable inline Heading extension. [#37386]
+- Subscribe block: Adds button-only style. [#37341]
+- Subscribe block: Allow inside navigation block. [#37439]
```

Comment:
Hmm... shouldn't it be 'navigation inside the block' instead of 'inside navigation block' ?

Reply:
In this case it literally is referring to adding the Subscription block as a child of the Navigation block (#37439). I tweaked it a bit in 8ad72ba but open to suggestions!

Reply:
Oh, I see, yeah, I think it makes more sense now. Could eventually maybe write it out to something like "Allow to be placed inside the Navigation block" or something. 👍

Reply:
Yeah, my first iteration was "Allow insertion in Navigation block." but that sounded too technical. Your version is pretty good; we can use that in the next release.